### PR TITLE
Enable issue details editing

### LINKED ISSUE #1608: Allow editing of issue details when creating from code action
When I create an issue from a TODO, I don't know that the comment would be exactly what I would want in the issue. It would be nice to get a details screen to add or remove details from the issue before it was submitted. 

Perhaps this is something that could be turned off with a setting in case people don't want the friction.

### PR DESCRIPTION
**Pull request by alexr00**
Fixes #1608

### CODE REVIEW

File: src/issues/issueFeatureRegistrar.ts [hunk 1]
```diff
@@ -61,6 +66,29 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
...
+		if (!vscode.window.activeTextEditor) {
+			return;
+		}
+		text = vscode.window.activeTextEditor.document.getText();
+		const indexOfEmptyLine = text.indexOf('\n\n');
+		if (indexOfEmptyLine < 0) {
+			return;
+		}
+		const title = text.substring(0, indexOfEmptyLine);
+		const body = text.substring(indexOfEmptyLine + 2);
```
Comment (RMacfarlane) [inline src/issues/issueFeatureRegistrar.ts:83]: should we trim off the explanation comment from the issue body?

File: src/issues/issueFeatureRegistrar.ts [hunk 2]
```diff
@@ -61,6 +66,29 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
...
+	async createIssueFromFile() {
+		if (this.createIssueInfo === undefined) {
+			return;
+		}
+		let text: string;
+		if (!vscode.window.activeTextEditor) {
+			return;
+		}
+		text = vscode.window.activeTextEditor.document.getText();
+		const indexOfEmptyLine = text.indexOf('\n\n');
```
Comment (RMacfarlane) [inline src/issues/issueFeatureRegistrar.ts:78]: is it at all possible that the line ending would be changed on save or anything like that?

File: src/issues/issueFeatureRegistrar.ts (hunk 1 above)
Comment (alexr00) [reply src/issues/issueFeatureRegistrar.ts:93]: I would leave it, similar to how GitHub leaves the comment in the issue templates.

File: src/issues/issueFeatureRegistrar.ts (hunk 2 above)
Comment (alexr00) [reply src/issues/issueFeatureRegistrar.ts]: Yes, added some handling for that.

**Pull request merged**